charts/k8tz/values.yaml

```diff
@@ -5,6 +5,7 @@ replicaCount: 1
 
 namespace: k8tz  # set to `null` to use helm built-in namespace
 createNamespace: true  # set to `false` to disable namespace creation by the helm chart
+namespaceLabels: {}
```

Comment:
please document the new option in chart readme: https://github.com/k8tz/k8tz/tree/master/charts/k8tz#values